docs/setup_docs.sh

```diff
@@ -59,7 +59,7 @@ if [ "$SKIP_INTEGRATE_CONNECTOR_DOCS" = false ]; then
   integrate_connector_docs rabbitmq v3.0
   integrate_connector_docs gcp-pubsub v3.0
   integrate_connector_docs mongodb v1.2
-  integrate_connector_docs opensearch v1.1
+  integrate_connector_docs opensearch v1.2
```

Comment:
The PR says 1.2.0 and 2.0.0 but I only see changes for 1.2.0. How about 2.0.0?

Reply:
docs are same for both

also a question: will integration docs support multiple branches per one connector name?

Reply:
since the only difference in functionality for opensearch connectors between v1.2.0 and v2.0.0 is the fact that one works with OpenSearch v1 and the other with v2 I don't think we need to split them.
So both v1.2 and v2.0 are ok here however need to pick only one

WDYT?